compose/bundle.py

```diff
@@ -108,12 +143,15 @@ def get_image_digest(service):
     if not digest:
         raise ValueError("Failed to get digest for %s" % service.name)
 
+    repo = parse_repository_tag(service.options['image'])[0]
     identifier = '{repo}@{digest}'.format(repo=repo, digest=digest)
 
     # Pull by digest so that image['RepoDigests'] is populated for next time
     # and we don't have to pull/push again
     service.client.pull(identifier)
 
+    log.info("Stored digest for {}".format(service.image_name))
```

Comment:
It'll still be necessary on pre-1.12 engines (although said engines don't support bundles, so that's arguably a minority case).